### PR TITLE
[WIP] I see dead code

### PR DESCRIPTION
**Pull request by shiftkey**
Wrapping my head around the code, deleting unnecessary things as I go

TODO: 
- [ ] namespaces in `Shimmer.WixUiClient` project are `Shimmer.Client.WixUI` - can we be consistent? which one wins?

### CODE REVIEW

File: src/ShimmerIAppUpdateTestTarget/TestAppSetup.cs [hunk 1]
```diff
@@ -81,7 +80,7 @@ public void OnVersionInstalled(Version versionBeingInstalled)
         public void OnVersionUninstalling(Version versionBeingUninstalled)
         {
             if (shouldThrow()) {
-                Console.WriteLine(((String) null).Length);
+                Console.WriteLine(((String) null).Length); // XXX: this seems wrong
```
Comment (anaisbetts) [inline src/ShimmerIAppUpdateTestTarget/TestAppSetup.cs:83]: I suspect it's part of the test where we verify what happens when the IAppUpdate fucks up. So in some sense, it _is_ wrong, but that's right :)

File: src/SampleUpdatingApp/App.xaml.cs [hunk 2]
```diff
@@ -1,17 +1,8 @@
-﻿using System;
```
Comment (anaisbetts) [inline src/SampleUpdatingApp/App.xaml.cs:1]: Never remove `System`, no matter how hard R# wants you to, hate that shit

Comment (shiftkey) [inline src/SampleUpdatingApp/App.xaml.cs:1]: Noted.

VS does the same behaviour with "Remove and Sort Usings"

File: src/ShimmerIAppUpdateTestTarget/TestAppSetup.cs (hunk 1 above)
Comment (shiftkey) [inline src/ShimmerIAppUpdateTestTarget/TestAppSetup.cs:83]: Well can it be a less obscure exception then?

Comment (anaisbetts) [inline src/ShimmerIAppUpdateTestTarget/TestAppSetup.cs:83]: That's reasonable, though I wanted it to simulate a real programmer fuckup

**Comment by ghost** [comment]
Shimmer: It's like ClickOnce but Works.

Please did they are any clear documentation about how do we use it ??!!

**Pull request closed without merging**